app/templates/package.json

```diff
@@ -2,19 +2,25 @@
   "name": "<%= moduleName %>",
```

Comment:
about renaming, does "package.json" working properly? when I tried it, it failed